scripts/trust-keys.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cd "$(dirname "$0")"
+
+export PASSWORD_STORE_DIR=../.private/passwords
+echo "Expecting passwords under $PASSWORD_STORE_DIR"
+
+while IFS= read -r key && [[ -n "$key" ]]; do
+  # fetch if not already there
+  gpg --list-public-key "$key" > /dev/null || gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys "$key"
```

Comment:
throwing it out there, might also be worth falling back to the system's default keyserver as a final fallbcak:

```suggestion
  gpg --list-public-key "$key" > /dev/null ||
    gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys "$key" ||
    gpg --recv-keys "$key"
```